src/app/components/modules/simplified-lunch/simplified-lunch.component.scss

```diff
@@ -0,0 +1,269 @@
+@import '../../../../styles/settings';
+
+/* --------------------------
+  Main Styles
+--------------------------- */
+.module-container{
+  width:100%;
```

Comment:
Much of the indentation here is in spaces. Could you convert all of the indentation to tabs 😉 ?